packages/@uppy/transloadit/src/Client.js

```diff
@@ -101,6 +126,13 @@ module.exports = class Client {
   cancelAssembly (assembly) {
     const url = assembly.assembly_ssl_url
     return fetch(url, { method: 'delete', headers: this._headers })
```

Comment:
maybe it's worth wrapping the `fetch` module into a reusable module that handles the `NetworkError` internally? This pattern is now spread across the codebase, and the need to change this snippet at one place will most likely imply the need to change it everywhere else.

Reply:
this sounds good—maybe something like `@uppy/utils/lib/fetch`